src/libraries/System.Text.Json/src/System/Text/Json/Serialization/Metadata/JsonTypeInfo.cs

```diff
@@ -966,7 +966,7 @@ internal static JsonTypeInfo CreateJsonTypeInfo(Type type, JsonConverter convert
             {
                 Type jsonTypeInfoType = typeof(JsonTypeInfo<>).MakeGenericType(type);
                 jsonTypeInfo = (JsonTypeInfo)jsonTypeInfoType.CreateInstanceNoWrapExceptions(
-                    parameterTypes: new Type[] { typeof(JsonConverter), typeof(JsonSerializerOptions) },
+                    parameterTypes: [typeof(JsonConverter), typeof(JsonSerializerOptions)],
```

Comment:
Another array that could be cached if it's worthwhile.

Reply:
It's not a hot path method and I feel that caching to a static would sacrifice readability in this case.